src/bank/bft_bank_net.rs

```diff
@@ -59,18 +63,21 @@ mod tests {
             let actor = net.initialize_proc();
             net.on_proc_mut(&actor, |p| p.trust_peer(genesis_actor));
             net.anti_entropy();
-            net.run_packets_to_completion(net.on_proc(&actor, |p| p.request_membership()).unwrap());
+	    let packets = net.on_proc_mut(&genesis_actor, |p| p.request_membership(actor).unwrap()).unwrap();
```

Comment:
more unwraps().    see my general comment above.  I won't comment on further instances.

Reply:
This code is all just for test cases, I hope it's fine to leave the unwraps here